components/Formats/index.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component, Fragment } from 'react'
+import { graphql, compose } from 'react-apollo'
+import { css } from 'glamor'
+import { descending } from 'd3-array'
+import gql from 'graphql-tag'
+import Loader from '../../components/Loader'
+import Link from '../Link/Href'
+import withT from '../../lib/withT'
+
+import {
+  Center,
+  TeaserFeed,
+  Editorial,
+  Interaction,
+  RawHtml,
+  mediaQueries
+} from '@project-r/styleguide'
+
+const styles = {
+  container: css({
+    padding: '30px 15px 120px',
+    [mediaQueries.mUp]: {
+      maxWidth: '695px',
+      padding: '60px 0 120px'
+    }
+  }),
+  h1: css({
+    marginBottom: '20px',
+    [mediaQueries.mUp]: {
+      marginBottom: '40px'
+    }
+  }),
+  h2: css({
+    marginBottom: '20px',
+    [mediaQueries.mUp]: {
+      marginBottom: '30px'
+    }
+  })
+}
+
+const getFormats = gql`
+  query getFormats {
+    documents(template: "format", feed: true) {
+      nodes {
+        meta {
+          kind
+          title
+          description
+          path
+          color
+          publishDate
+        }
+      }
+    }
+  }
+`
+
+class Formats extends Component {
+  render() {
+    const { data: { loading, error, documents }, t } = this.props
+    return (
+      <Loader
+        loading={loading}
+        error={error}
+        render={() => {
+          const editorialNodes = documents
+            ? [...documents.nodes]
+                .filter(node => node.meta.kind === 'editorial')
+                .sort((a, b) =>
+                  descending(a.meta.publishDate, b.meta.publishDate)
+                )
+            : []
+          const metaNodes = documents
+            ? [...documents.nodes]
+                .filter(node => node.meta.kind === 'meta')
+                .sort((a, b) =>
+                  descending(a.meta.publishDate, b.meta.publishDate)
+                )
+            : []
+          return (
+            <Center {...styles.container}>
+              <Interaction.H1 {...styles.h1}>
+                {t('formats/title')}
+              </Interaction.H1>
+              <RawHtml
+                type={Editorial.P}
+                dangerouslySetInnerHTML={{
+                  __html: t('formats/lead')
+                }}
+              />
+              {!!editorialNodes.length && (
+                <Fragment>
+                  <Interaction.H2 {...styles.h2}>
+                    {t('formats/title/editorial')}
+                  </Interaction.H2>
+                  {editorialNodes.map(doc => (
+                    <TeaserFeed {...{...doc.meta, kind: 'meta'}} Link={Link} key={doc.meta.path} />
```

Comment:
`<TeaserFeed {...doc.meta} kind='meta'` would be cleaner.

However we should also be consistent with this behaviour (have the same in the main feed).

Possible solution:
- Add a template prop to `TeaserFeed`
- expend the [interaction headline clause](https://github.com/orbiting/styleguide/blob/11a30d52a2107ff2a127983c2c4cfea4a2fbc510/src/components/TeaserFeed/index.js#L53) with `template === 'format'`
- query template in `pages/feed`, hard code template='format' here or also query above